tests/suite/scale_test.go

```diff
@@ -716,18 +708,16 @@ var _ = Describe("Scale test", Ordered, Label("nfr", "scale"), func() {
 				"header-50": {"header-50-val"},
 			},
 		})
-
-		Expect(resourceManager.DeleteFromFiles(matchesManifests, ns.Name)).To(Succeed())
 	})
 
 	AfterEach(func() {
 		teardown(releaseName)
+		Expect(resourceManager.Delete([]client.Object{ns})).To(Succeed())
```

Comment:
One issue I found while writing multiple tests is that the namespace wouldn't be fully terminated before the next test started and tried to create it again, and it would fail. I needed to add a wait after deletion (similar to when we tear down NGF in the suite).

I also found an issue where I needed to change the top level namespace definition to a non-pointer value, because it would become populated with a `resourceVersion` when you `Get` it, and then you can't re-create it otherwise the k8sClient throws an error.